tox.ini

```diff
@@ -4,6 +4,11 @@ envlist=py27,py26,py33,py34,py35
 
 [testenv]
 commands=
-  pytest --junitxml={envlogdir}/junit-{envname}.xml []
+  pytest
```

Comment:
```suggestion
  pytest []
```

Reply:
I changed it to `{posargs}` now if you don't mind, I was sure the `[]` was some sort of typo...